jetty-servlet/src/main/java/org/eclipse/jetty/servlet/DefaultServlet.java

```diff
@@ -420,8 +419,7 @@ else if (_servletContext instanceof ContextHandler.Context)
             }
             else
             {
-                URL u = _servletContext.getResource(pathInContext);
-                r = _contextHandler.newResource(u);
```

Comment:
Why was this changed? Can `_servletContext` never be other than `ContextHandler.Context`?